src/http.rs

```diff
@@ -977,6 +981,7 @@ fn vrps(
     Response::builder()
     .header("Content-Type", format.content_type())
     .header("content-length", stream.output_len())
+    .header("Last-Modified", done.unwrap().format_with_items(http_date.clone()).to_string())
```

Comment:
While unwrapping here should be fine here right now, it is probably still better to only add the header when it is actually available. Maybe you can wrap it in an `if let Some(done) = done { }` block somehow?

Reply:
Thanks for the feedback.

The issue is: this is the first time ever I'm trying to code in Rust, and I have to say that the documentation is not simple to understand.

I'll try this though.

Reply:
You probably have to break it up into three parts using variables.

You could either use a mutable variable and reassign to it:

```
let mut builder = Response::builder();
if let Some(done) = done {
    builder = builder.header( /* ... */);
}
builder.more_stuff()
```

Or, the move the value along to a new variable (re-using the same name which is fine):

```
let builder = Builder::builder();
let builder = if let Some(done) = done {
    builder.header( /* ... */)
} else {
    builder
};
builder.more_stuff()
```

The latter is probably more proper but also more wordy, so I would probably go for the former.